content/docs/next-release/operator.md

```diff
@@ -334,13 +334,61 @@ The default create-schema job uses `MODE=prod`, which implies a replication fact
 
 ### Elasticsearch storage
 
+By default Elasticsearch storage does not require to run any initialization job. However Elasticsearch
+storage requires to run a cron job to clean old data from the storage. 
+ 
+When rollover (`es.use-aliases`) is used Jaeger operator also deploys a job to initialize Elasticsearch storage
```

Comment:
"used" -> "enabled, "
"a job" -> "a cron job"

Reply:
The initialization is done by a job and not a cron job

Reply:
Ah ok :)